web/static/css/app.css

```diff
@@ -265,6 +292,31 @@ footer {
   margin: 1.2em -1.2em -1.2em -1.2em;
 }
 
+.team-management {
+  padding: 0px 150px;
+  color: #31b0e2;
+}
+
+.team-management h2 {
+  padding: 40px 0 20px 0;
+}
+
+.preferences {
+}
+
+.preferences__input--inline {
```

Comment:
The thing after the double dashes is a Modifier (the M in BEM), so you would normally have base styles under `.preferences__input` and a style modifier under `.preferences__input--inline`.